src/scss/_components.hero.scss

```diff
@@ -222,17 +261,19 @@ $z-heading: 600;
       display: block;
       position: absolute;
       @include positions(0);
-      background: url("/images/lightning-dim.png") no-repeat top right scroll / auto 100%;
+      background: url("/images/lightning-dim.png") no-repeat top right;
+      background-size: auto 100%;
     }
 
-    @supports (mix-blend-mode: screen){
+    @supports (mix-blend-mode: overlay){
```

Comment:
Should this be `screen`? Just wondering since we're not using `overlay` in this code block.